### DISCUSSION TITLE
3 - New Ideas

### DISCUSSION THREAD

**Discussion by SAGEof6iixPATHS** [hahwul/dalfox#185] [Ideas]
Add a option to skip the url scanning on...
Add a option to pause the scan and resume it like ctrl+c and if run that same command on that file of urls then it should start from where is was stopped...
Add a option to skip to url number.... Like i attached a 100 urls then i will insert 20 and then it will start scanning from the 20th url...

I know that is a big task that i just gave you but when i use the script every time i think that this should be there so i told ya!!

**Comment by hahwul** [comment]
Hi @SAGEof6iixPATHS 
I also recognize that it is a necessary function. I'll roll up sleeves
Thank you :D

**Reply by hahwul** [reply]
issue number : https://github.com/hahwul/dalfox/issues/187
